engine/settings/prod_without_db.py

```diff
@@ -13,7 +15,6 @@ def on_uwsgi_worker_exit():
     # Only works under uwsgi web server environment
     pass
```

Comment:
this isn't at all related to your PR, but I don't quite understand this block between lines 3 - 16. I don't think it's _actually_ doing anything. I _think_ the original intent may have been this:
```python
import uwsgi
from prometheus_client import multiprocess

def on_uwsgi_worker_exit():
  multiprocess.mark_process_dead(os.getpid())

uwsgi.atexit = on_uwsgi_worker_exit # note how this is unindented, if it's nested within the function definition it doesn't do anything as this function is never called
```